engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/stores/ValueStore.scala

```diff
@@ -27,15 +27,10 @@ object ValueStore {
 case class ValueStore(store: Table[OutputPort, ExecutionIndex, WomValue]) {
 
   override def toString: String = {
-    import io.circe.syntax._
-    import io.circe.Printer
+    import common.util.StringUtil.EnhancedToStringable
 
-    val values = store.valuesTriplet.map {
-      case (node, None, value) => node.name -> value.valueString
-      case (node, index, value) => s"${node.name}:${index.fromIndex}" -> value.valueString
-    }.toMap
-
-    values.asJson.printWith(Printer.spaces2.copy(dropNullValues = true, colonLeft = ""))
+    // ValueStores can be large and have been known to cause OOMs during unbounded stringification
+    store.valuesTriplet.toPrettyElidedString(limit = 1000)
```

Comment:
This pretty fundamentally changes the structure of what gets printed, which might be fine, but presumably it was like that because it was helpful. I wonder whether we could keep the string structure while also applying a sanity limit? Maybe something along the lines of 
```
store.valuesTriplet.take(10).map {
      case (node, None, value) => node.name -> value.toPrettyElidedString(limit = 100)
      case (node, index, value) => s"${node.name}:${index.fromIndex}" -> value.toPrettyElidedString(limit = 100)
}
```

?

Reply:
(especially since this sounds like this OOM is hiding a more fundamental error in the value store which we may want to debug at some point in the near(?) future)

Reply:
This sounds great, I did not notice this opportunity on Friday in fix-it-now mode.